--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -42,7 +42,6 @@ install(PROGRAMS 
   cvs-clean 
   cvsblame 
   colorsvn 
-  svn-clean 
   svnlastchange 
   svnversions 
   kdesvn-build 
@@ -62,4 +61,6 @@ install(PROGRAMS 
   svnintegrate 
   kde_generate_export_header
   DESTINATION ${BIN_INSTALL_DIR})
- 
+
+install(PROGRAMS svn-clean DESTINATION ${BIN_INSTALL_DIR} RENAME svn-clean-kde)
+
