Index: kdesdk-4.0.98/scripts/CMakeLists.txt
===================================================================
--- kdesdk-4.0.98.orig/scripts/CMakeLists.txt	2008-07-10 21:01:55.000000000 +0000
+++ kdesdk-4.0.98/scripts/CMakeLists.txt	2008-07-10 21:02:13.000000000 +0000
@@ -44,7 +44,6 @@
   cvsblame 
   colorsvn 
   svn2dist 
-  svn-clean 
   svnlastchange 
   svnversions 
   kdesvn-build 
@@ -64,4 +63,6 @@
   svnintegrate 
   kde_generate_export_header
   DESTINATION ${BIN_INSTALL_DIR})
- 
+
+install(PROGRAMS svn-clean DESTINATION ${BIN_INSTALL_DIR} RENAME svn-clean-kde)
+
