Fixed-For: 4.5.0
Index: kdesdk-4.3.0/CMakeLists.txt
===================================================================
--- kdesdk-4.3.0.orig/CMakeLists.txt	2009-08-22 00:52:14.000000000 +0200
+++ kdesdk-4.3.0/CMakeLists.txt	2009-08-22 00:52:50.000000000 +0200
@@ -51,11 +51,11 @@
 macro_optional_add_subdirectory(strigi-analyzer)
 macro_optional_add_subdirectory(kioslave)
 
-if (CMAKE_SYSTEM_NAME MATCHES Linux)
+if (CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD OR CMAKE_SYSTEM_NAME STREQUAL GNU)
   #kmtrace is highly non-portable and requires glibc
   #TODO: replace with a check for glibc
   macro_optional_add_subdirectory(kmtrace)
-endif (CMAKE_SYSTEM_NAME MATCHES Linux)
+endif (CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD OR CMAKE_SYSTEM_NAME STREQUAL GNU)
 
 macro_optional_add_subdirectory(kompare)
 macro_optional_add_subdirectory(kprofilemethod)
